standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/messaging/json/JSONAllocWriteIdMessage.java

```diff
@@ -29,26 +31,36 @@
 public class JSONAllocWriteIdMessage extends AllocWriteIdMessage {
 
   @JsonProperty
-  private String server, servicePrincipal, tableName;
+  private String server, servicePrincipal, dbName, tableName;
 
   @JsonProperty
-  private List<Long> txnIds;
+  private List<Long> txnIdList, writeIdList;
```

Comment:
Can't we store it as list of TxnToWriteId?

Reply:
json can not parse TxnToWriteId